server/src/main/java/org/apache/iotdb/db/engine/storagegroup/StorageGroupProcessor.java

```diff
@@ -979,6 +983,7 @@ private boolean insertTabletToTsFileProcessor(
 
     try {
       tsFileProcessor.insertTablet(insertTabletPlan, start, end, results);
+      writeLock();
```

Comment:
Why do you add writeLock here? We have get write lock before call the function.

Reply:
Yes, it is a mistake......